Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -2106,9 +2106,13 @@ private extension Purchases {
         ) { [cache = self.paywallCache] offeringsResultData in
             if #available(iOS 15.0, macOS 12.0, watchOS 8.0, tvOS 15.0, *),
                let cache = cache, let offerings = offeringsResultData.value?.offerings {
-                self.operationDispatcher.dispatchOnWorkerThread {
-                    await cache.warmUpEligibilityCache(offerings: offerings)
-                    await cache.warmUpPaywallImagesCache(offerings: offerings)
+                await withTaskGroup(of: Void.self) { group in
+                    group.addTask {
+                        await cache.warmUpEligibilityCache(offerings: offerings)
+                    }
+                    group.addTask {
+                        await cache.warmUpPaywallImagesCache(offerings: offerings)
+                    }
                 }
```

Comment:
I assume we'll add fonts here too, in parallel?

I wondering if it wouldn't be better to have one task per image, eligibility, or font, instead of parallelizing the method. I say this because inside fonts we're launching yet another task group